test/lib/mayaUsd/utils/materialx_test_data/topology_tests_1_39.mtlx

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0"?>
+<materialx version="1.38" colorspace="lin_rec709">
```

Comment:
Should this say 1.39?
```suggestion
<materialx version="1.39" colorspace="lin_rec709">
```

Reply:
Yes, and no. It does not contain any node that changed between versions, so it is safe to stay at 1.38.

Reply:
I see! Seeing `1_39` in the filename and `version="1.38"` in the document looked like a typo, but that makes sense.